viz_config.yml

```diff
@@ -31,7 +31,7 @@ hurricane_col_names: ["Tropical Storm %s", "Hurricane %s (Cat1)", "Hurricane %s
 storm_line_col: "grey30"
 major_river_col: "#1874cd"
 gage_river_col: "#5d9ddc"
-lake_col: "#a2c7eb"
+lake_col: "#EBF6F8"
```

Comment:
want this over in the main repo?

Reply:
I prefer the previous setting where lakes were a different color from the ocean. When the great lakes were added to this one, the polygon shapes were cut in half (US only boundaries I guess?) and the ocean color permeated through. It was easier to match them in color than to add different geometries